### ISSUE TITLE
Sort problem?.

### ISSUE THREAD

**Issue by CarlosThinkBig** [h2oai/datatable#1475]
Hi,

I am using `datatable version 0.7.0`.
Working with these examples I do not see that `sort` method is working:

```Python
np.random.seed(1)
mydtb = dt.Frame({"A" : 10*np.random.randn(1000), "B" : 10*np.random.randn(1000), "C": 10*np.random.randn(1000), "D": 10*np.random.randn(1000) })
```
that produces this output:
```python
0	16.2435	−1.53236	4.89517	−0.77107
1	−6.11756	−24.3251	2.38796	2.07825
2	−5.28172	5.07984	−4.48112	9.86196
3	−10.7297	−3.24032	−6.10795	14.3276
4	8.65408	−15.1108	−20.2995	5.28258
5	−23.0154	−8.71422	6.07947	−3.67732
6	17.4481	−8.6483	−3.54109	6.91721
7	−7.61207	6.08749	1.52581	−7.98347
8	3.19039	5.61638	5.01275	2.13818
9	−2.4937	15.1475	−7.86403	17.8998
```
and trying to `sort`...
(with the output I get in the Jupyter notebook I can not copy&paste the names of the columns but the first one is "A").

```python
mydtb.sort("A")
```
produces this ouput, which has the same order :

```python
0	16.2435	−1.53236	4.89517	−0.77107
1	−6.11756	−24.3251	2.38796	2.07825
2	−5.28172	5.07984	−4.48112	9.86196
3	−10.7297	−3.24032	−6.10795	14.3276
4	8.65408	−15.1108	−20.2995	5.28258
5	−23.0154	−8.71422	6.07947	−3.67732
6	17.4481	−8.6483	−3.54109	6.91721
7	−7.61207	6.08749	1.52581	−7.98347
8	3.19039	5.61638	5.01275	2.13818
9	−2.4937	15.1475	−7.86403	17.8998
```

I repeated the same example, but just with a one column dataframe but the results were exactly the same.

Thanks,
Carlos.

**Comment by st-pasha** [comment]
Hi Carlos,
What you're seeing is an instance of bug #1448, which was resolved some 2 weeks ago, but after 0.7.0 was published. With the latest dev version of datatable, it works as expected:

<img width="422" alt="screen shot 2018-12-11 at 11 43 57 am" src="https://user-images.githubusercontent.com/4231472/49825952-88367180-fd3a-11e8-864f-e30543cf8e1f.png">

To be clear, the issue that you're seeing are not problems with sorting, but rather with displaying the Frames in a Jupyter notebook.

**Comment by CarlosThinkBig** [comment]
Thanks and sorry for the confusion.
Carlos.
